packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -44,6 +45,7 @@ export const ParticipantListWithHeading = (props: {
   title?: string;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  onParticipantClick?: (props?: ParticipantListParticipant) => void;
```

Comment:
is `onParticipantClick` a `ParticipantListProps` now? i.e. should onParticipantClick exist inside participantListProps and not be a seperate prop here